linux/docker_virtualenv_ius_nginx.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e -u -x
+
+cat << EOF > /etc/yum.repos.d/nginx.repo
+[nginx]
+name=nginx repo
+baseurl=http://nginx.org/packages/centos/\$releasever/\$basearch/
+gpgcheck=0
+enabled=1
+EOF
+
+OMEROVER=omero
+#OMEROVER=omerodev
+
+source settings.env
+
+#install nginx
+yum -y install nginx
+
+#Create virtual env.
+virtualenv /tmp/omeroenv
+set +u
+source /tmp/omeroenv/bin/activate
+set -u
+
+# install omero dependencies
+/tmp/omeroenv/bin/pip2.7 install pillow numpy matplotlib
+
+# Django
+/tmp/omeroenv/bin/pip2.7 install "Django<1.9"
+
+/tmp/omeroenv/bin/pip2.7 install gunicorn
```

Comment:
That's an option but we will have to "break" step04 since ```/home/omero/OMERO.server/...``` does not exist when the dependencies are installed.